test/uk/gov/hmrc/transitmovementsrouter/generators/TestModelGenerators.scala

```diff
@@ -123,4 +131,49 @@ trait TestModelGenerators extends BaseGenerators {
     )
   }
 
+  implicit def arbitrarySdesResponse(): Arbitrary[SdesNotificationItem] = Arbitrary {
```

Comment:
as we are not passing any parameter with this def so please make it **val** instead of **def**